### PR TITLE
`simulatedWorkspaceOutages` should be JSON serializable

### PR DESCRIPTION
**Pull request by mxsdev**
This is breaking the fake quick server

cc @xplato

### CODE REVIEW

File: src/lib/database/store.ts [hunk 1]
```diff
@@ -418,16 +418,16 @@ const initializer = immer<Database>((set, get) => ({
...
         routes,
-      })
+      }
     })
   },
 
   simulateWorkspaceOutageRecovery(workspace_id) {
     set((state) => {
-      state.simulatedWorkspaceOutages.delete(workspace_id)
+      state.simulatedWorkspaceOutages[workspace_id] = undefined
```
Comment (razor-x) [inline src/lib/database/store.ts:430]: Can this be deleted from the object instead of set undefined?

**Review by razor-x — APPROVED** [review]

**Review by seveibar — APPROVED** [review]
Agree that deletion is a bit better

File: src/lib/database/store.ts (hunk 1 above)
Comment (mxsdev) [reply src/lib/database/store.ts:430]: There is a lint rule disallowing deletion

Comment (seveibar) [reply src/lib/database/store.ts:430]: This bug is blocking some people so I created an issue to discuss the lint rule and "fixing" the deletion

https://github.com/seamapi/fake-seam-connect/issues/83

**Pull request merged**